students/johnpharmd/lesson05/mailroom3.py

```diff
@@ -22,7 +20,6 @@
 
 # Processing
 def send_ty():
-    # global donors_amts
```

Comment:
I would recommend re-visiting this function (`send_ty`)

here are some main points:
* list comprehensions are used to collect data (it has return value) so using it for print statements or to see if exception is thrown isn't really the right pattern
* it took me a second to understand the reason for `ZeroDivisionError`. Simple is usually better, my understanding is that you're checking to see if there are any numbers in the name? use `response.isalpha()` instead.
* u should avoid things like lines 27 and 42. if you're having to write duplicate code it is probably needing a loop there. You can do while True and exit when the input you got is good
* there are too many nested if / else blocks which makes it hard to read to understand the logic. 
* remove commented out code, it makes it harder to read